public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.utils.ts

```diff
@@ -192,3 +192,6 @@ export const getInitialValues = (alertRule: AlertRule): AddAlertRuleFormValues =
     threshold: formatEditThreshold(params),
   };
 };
+
+export const minValidator = (min: number) => (value: any): undefined | string =>
+  value >= min ? undefined : `Must be greather than or equal to ${min}`;
```

Comment:
JFYI: since you have access to the inner HTML input element, I think you could just pass a `min` prop to it, instead of using a validator

Reply:
I thought the same here but I tried this and in some browsers (e.g. firefox) the min doesn't work that well so I would keep this as safe guard.

Reply:
@nicolalamacchia Yes, I'm passing that in `AddAlertRulesModal` as well. The problem is that with only that, the button will still be enabled. Once you click it, you'll see a native warning on the input, which would be very different from the error labels we have in place now.

Reply:
yes, it's still better to have our own validator